internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -609,6 +609,18 @@ func convertEntryPoint(entrypoint manifest.EntryPointOverride) ([]string, error)
 	return out, nil
 }
 
+func getDeployConfig(deployStrategyName string) template.DeploymentConfigurationOpts {
```

Comment:
tiny tiny nit: 
```suggestion
func convertDeploymentConfig(deployStrategyName string) template.DeploymentConfigurationOpts {
```

On a separate note, what do you think of this signature:
```convertDeploymentConfig(deploymentConfig manifest.DeploymentConfiguration) template.DeploymentConfigurationOpts```
?